### ISSUE TITLE
Want to ask something about ”dream“

### ISSUE THREAD

**Issue by aboutmydreams** [holman/feedback#865]
I am from China. I am currently a junior undergraduate at Nanchang University. Wen Hao, I searched for "ask me anything" on GitHub and saw you on github.
I do n’t know if you sometimes have the feeling of seeing the end of life at first glance. I feel very strongly recently. Engaging in a more humane and creative education is my dream since junior high school.
The hatred of education and the improvements it wants to make, but now.
The products that people want to use are gradually formed after they came to the university to enter the studio as PMs. Today, they have indeed made some products that can bring convenience to other students.
Yesterday I came back home from school to think about life. From the perspective of achieving dreams, goals should be descriptive and can be broken down into small goals. However, when I tried to break down how to create an ideal school,
I do n’t know how to decompose it. There seems to be no way for me to start a school, and the students that I nurture can be recognized by the society, because there are a lot of problems with this current social standard. Diplomas are still so important today.
Although the abilities of students are almost the same or even the opposite, what can change social prejudice? It seems that as long as education itself has the greatest influence on the formation of social prejudice, I gradually began to wonder: Is it possible to realize dreams?
I realized that my dream is gradually dying, and I hope that the senior can point me in the direction. I also want to know the status of the realization of the senior's dream.
Grateful!

**Comment by holman** [comment]
Dreams are definitely still around, and still attainable!

I don’t think I can give you a concise answer with all of this, since it deals with a tonnnn of different things: the role of education in our lives (especially as our tech industry matures), feelings of burnout/depression, how we progress through our careers, and so on.

I guess overall I’d say a few things: first, people change. Especially yourself. Who I was when I was in school versus who I am now... well, they’re really, really, different people. My viewpoints and things I actually cared deeply about ten years ago compared to today is wayyyyy different. And that’s a good thing. Some things and passions I miss, but I’m still happier with my viewpoint on life today.

So basically, what you feel today doesn’t necessarily mean you’ll feel that way in the future, either.

Secondly, the role of education... yeah, that’s been something people have struggled with for generations. I have a lot of thoughts on it too, but in general I think education is what you make of it. I went to a good school, but almost everything of what I learned while at school is not what I use in my normal day today. But I still learned a lot of things on my own which lead to where I am today.

In terms of changing it for more than just yourself? Yeah, that’s hard. I think the world does need more different ways of thinking when it comes to education, and new schools and startups are a huge part of that. But I also wouldn’t necessarily get caught up in doing that right now- sometimes solutions might come to you as you learn more and experience more in life.

But yeah, everyone’s different. I hope you the best, and I hope you find what you’re looking for. :)
